twilio/twiml/messaging_response.py

```diff
@@ -7,7 +7,7 @@ class MessagingResponse(TwiML):
     """
     def __init__(self):
         """
-        Create a new <Response>
+        Create a new <MessagingResponse>
```

Comment:
`str(MessagingResponse())` returns `<Response>`, but I'm not sure if we want to be general or specific.